deploy/deploy_citus_server.yml

```diff
@@ -39,6 +39,21 @@
         src: "{{ pp_file_transfer_dir }}"
         state: link
 
+    - name: Check WAL directory exists
+      stat:
+        path: "{{ pp_xlogdir }}/pg_wal"
```

Comment:
@rioh We should probably make "pg_wal" a variable, in case we need to deploy Postgres 9.x for some reason.